### DISCUSSION TITLE
[Question] Modifing a Theme

### DISCUSSION THREAD

**Discussion by EliteAMDGamer** [tsujan/Kvantum#953] [Q&A] [answered]
I am currently trying to modify the [mac Cheetah theme](https://www.pling.com/p/1686250)  to that the background is a solid mac [off white](tps://www.pling.com/p/1001792/) since there are not leopard style themes. I've modified all the  boxes with the stripe pattern in the svg file and change the file name but no luck. Any help would be appreciated. As I want aqua well a non flat theme that's not dark to go with my kde oxygen air like theme 

Here is what I mean by stripes an background.

![kvantum](https://github.com/tsujan/Kvantum/assets/76790811/10001056-5ae3-44d8-818e-845d0179da5b)

**Comment by tsujan** [comment, marked as answer]
This is rather a downstream question.

> I've modified all the boxes with the stripe pattern in the svg file

No need to change the SVG file. Inside the `kvconfig` file, remove the keys with `patternsize` in their names from the sections `[Dock]`, `[Toolbar]`, `[TabFrame]`, `[Menu]`, `[MenuBar]` and `[WindowTranslucent]`, but not from other sections.

Since the SVG element of `tlbar` has also a sort of interior pattern, rename it to `tlbar1` everywhere inside the `kvconfig` file.

All the needed info is in the documentation — consult the doc folder inside the source.

EDIT: I assumed your question was "How to remove background patterns from this theme?"

**Reply by EliteAMDGamer** [reply]
Yes, thank you so much I'm pretty new to this. I apologize for the miss placement of the question.
